docs/source/contributing/install-nodejs.md

```diff
@@ -0,0 +1,12 @@
+1.  Install or update the supported LTS versions of Node.js, then activate the version supported in Volto.
+
+    ```shell
+    nvm install "lts/*"
+    nvm use 20
```

Comment:
It looks like we can suggest:
```
nvm install --lts
nvm use --lts
```
and then we don't need to update this when we start supporting a new LTS version.

Reply:
Good idea. Done in 7f52aa0733a84207ad7f9e3c072c4d8c088b22b0

Reply:
One thing that still bugs me is the Step 2:

```shell
$ nvm use --lts
Now using node v20.10.0 (npm v9.8.1)

$ node -v      

v20.6.1
```

I have no clue why that is different.

Reply:
@stevepiercy Maybe you have a PATH configured that is finding a different copy of Node somewhere else instead of the one installed by nvm?

Reply:
Here's what I found:

```shell
$ tree .nvm/versions/node -L 1
.nvm/versions/node
├── v12.16.1
├── v12.16.2
├── v13.12.0
├── v15.11.0
├── v15.2.1
├── v16.13.2
├── v16.14.0
├── v16.17.1
├── v16.18.1
├── v16.19.0
├── v16.20.2
├── v17.5.0
├── v17.6.0
├── v18.12.1
├── v18.18.0
├── v20.10.0
└── v20.9.0

$ which node
/opt/homebrew/bin/node

$ node -v      

v20.6.1

$ brew uninstall node
Uninstalling /opt/homebrew/Cellar/node/20.6.1... (2,398 files, 57.5MB)

$ node -v
v20.10.0
```

I must have used Homebrew on my old laptop and it migrated over. Thanks for the nudge.

There's still this issue that I don't know how to resolve:

https://github.com/plone/volto/issues/5556#issuecomment-1864804427